zuora.gemspec

```
@@ -1,36 +1,30 @@
-# -*- encoding: utf-8 -*-
-$:.push File.expand_path("../lib", __FILE__)
-require "zuora/version"
+# frozen_string_literal: true
+
+$:.push File.expand_path('../lib', __FILE__)
+require 'zuora/version'
 
 Gem::Specification.new do |s|
-  s.name        = "zuora"
+  s.name        = 'zuora'
   s.version     = Zuora::Version.to_s
-  s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Josh Martin"]
-  s.email       = ["josh.martin@wildfireapp.com"]
-  s.homepage    = "http://github.com/wildfireapp/zuora"
-  s.summary     = %q{Zuora - ActiveModel backed client for the Zuora API}
-  s.description = %q{Zuora - Easily integrate the Zuora SOAP API using ruby objects.}
-  s.files         = `git ls-files`.split("\n")
-  s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
-  s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
-  s.require_paths = ["lib"]
-  s.extra_rdoc_files = [ "README.md" ]
+  s.authors     = ['Josh Martin']
+  s.email       = ['josh.martin@wildfireapp.com']
```

We should probably update the author and email.
```suggestion
  s.authors     = ['Sage Accounting']
  s.email       = ['sageone@sage.com']
```